spec/lib/finder_api_spec.rb

```diff
@@ -49,7 +49,7 @@ def result_item(id, title, score:, popularity:, updated:)
     end
 
     context 'updated-newest' do
-      subject { described_class.new("/finder", {}, order: 'updated-newest').content_item_with_search_results }
+      subject { described_class.new("/finder", 'order' => 'updated-newest').content_item_with_search_results }
```

Comment:
Same again with the lint errors in this file, can you fix them up to the previous commit.